docs/whats_new.rst

```diff
@@ -11,6 +11,42 @@ Changelog
 This page keeps a detailed human friendly rendering of what's new and changed
 in specific versions.
 
+
+.. _vp2p3p1:
+
+v2.3.1
+-------
+
+Bug Fixes
+~~~~~~~~~~
+
+- |commands| Fix Python 3.10 union type annotations in slash commands (:issue:`231`)
+- Fix double ``@`` when mentioning default role (:issue:`244`)
+- Fix a command sync bug which prevented registering two application commands with the same name but different types (:issue:`254`)
+- Fix :func:`GuildScheduledEvent.fetch_users`
+- Fix bug when editing a :class:`~ui.View` with URL buttons that resulted in the ``row`` attribute being reset (:issue:`252`)
+- |commands| Fix :class:`~ext.commands.MessageConverter` not defaulting to current channel if no channel ID specified (:issue:`284`)
+- Fix attachment descriptions not working after recent API change (:issue:`241`)
+- Fix incorrect type check in :func:`Guild.create_scheduled_event` using :class:`StagePrivacyLevel` instead of :class:`GuildScheduledEventPrivacyLevel`
```

Comment:
```suggestion
- Fix incorrect type check in :func:`Guild.create_scheduled_event` using :class:`StagePrivacyLevel` instead of :class:`GuildScheduledEventPrivacyLevel` (:issue:`263`)
```
Feel free to skip this if not necessary